README.md

```diff
@@ -6,416 +6,198 @@
 [![CI-Nightly](https://github.com/JuliaSIMD/Polyester.jl/actions/workflows/CI-julia-nightly.yml/badge.svg)](https://github.com/JuliaSIMD/Polyester.jl/actions/workflows/CI-julia-nightly.yml)
 [![Coverage](https://codecov.io/gh/JuliaSIMD/Polyester.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaSIMD/Polyester.jl)
 
-Note that `Polyester.@batch` moves arrays to threads by turning them into [StrideArraysCore.PtrArray](https://github.com/JuliaSIMD/StrideArraysCore.jl)s.
-This means that under an `@batch` slices will create `view`s by default, and that you may also need to start Julia with `--check-bounds=yes` while debugging.
+Polyester.jl provides **low-overhead multithreading** in Julia. The primary API is `@batch`, which can be used to parallelize for-loops (similar to `@threads`).
+
+Polyester implements static scheduling (c.f. `@threads :static`) and has minimal overhead because it manages and re-uses a dedicated set of Julia tasks. This can lead to (great) speedups compared to other multithreading variants (see [Benchmark]() below).
+
+## Basic usage example
 
-Polyester.jl provides low overhead threading.
-The primary API is `@batch`, which can be used in place of `Threads.@threads`.
-The number of available threads is still governed by [`--threads` or `JULIA_NUM_THREADS`](https://docs.julialang.org/en/v1.6/manual/multi-threading/#Starting-Julia-with-multiple-threads), as reported by `Threads.nthreads()`.
-Lets look at a simple benchmark.
 ```julia
-using Polyester, LinearAlgebra, BenchmarkHistograms
-# Single threaded.
-function axpy_serial!(y, a, x)
-    @inbounds for i in eachindex(y,x)
-        y[i] = muladd(a, x[i], y[i])
+using Polyester
+
+function axpy_polyester!(y, a, x)
+    @batch for i in eachindex(y,x)
+        y[i] = a * x[i] + y[i]
```

Comment:
Why no `muladd`?
Because Julia takes a strong stance against unexpectedly giving people better performance and accuracy, I'm pro-normalizing-`muladd`.

Zen CPUs probably don't `muladd` much faster than separate `mul` and `add`, because they have 2 adders and 2 multiply units that can also fma.
For the `muladd` version on Zen, the adders are idle, so throughput is the same w/ respect to the backend (but fewer instructions and uops with `muladd` is friendlier to the frontend).

Reply:
I'm not strongly opposed to using `muladd`, I just don't think it matters here. The purpose of this example is to demonstrate how to use `@batch`. Optimizing the axpy kernel itself is an orthogonal issue IMO and only a distraction.

Reply:
That's fine. FWIW, even on my intel system, `muladd` wasn't faster. `axpy` isn't compute bound.